packages/manager/modules/anthos/src/dashboard/controller.js

```diff
@@ -1,10 +1,26 @@
 export default class AnthosCtrl {
   /* @ngInject */
-  constructor($translate) {
+  constructor($translate, CucCloudMessage) {
     this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
   }
 
   $onInit() {
-    console.log('Hello', this.$translate.instant('world'));
+    this.messageContainer = 'anthos.dashboard';
+    this.loadMessages();
+  }
+
+  loadMessages() {
+    this.CucCloudMessage.unSubscribe(this.messageContainer);
```

Comment:
I guess we decided not to use the CuC components going forward? Please correct me if I ma wrong... I guess we can use `Alerter`

Reply:
I am not aware about this point.
May be i am off when you discussed on that.

Reply:
I will take into account next times.

Reply:
We will have to get it done now... Else, we will be using both `CucCloudMessage` and `Alerter`